changes.d/core/prompted-dependency-sources.md

```
@@ -0,0 +1,24 @@
+---
+links:
+  '#869': https://github.com/dahlia/optique/issues/869
+  '#870': https://github.com/dahlia/optique/issues/870
+  '#912': https://github.com/dahlia/optique/pull/912
+---
+ -  Fixed dependency sources completed by a prompt fallback to register
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)